common/factions/anb_factions.txt

```diff
@@ -676,3 +676,52 @@ mykx_xhazobkult = {
 		yearly_patriarch_authority = 0.02
 	}
 }
+
+########################################
+# Voltenu Factions
+########################################
+
+#Stakeholders
+voltenu_stakeholders = {
+	monarch_power = MIL
+	always = yes
+
+	modifier = {
```

Comment:
The modifiers are too strong for my taste

Reply:
Is now literally Technocracy Factions.